creator-node/test/prometheus.test.js

```diff
@@ -126,6 +133,29 @@ describe('test Prometheus metrics', async function () {
     assert.ok(!resp.text.includes('/content/:CID'))
   })
 
+  it('Checks that accessing /ipfs/:CID records explicit results', async function () {
+    await request(app).get('/ipfs/QmVickyWasHere')
+    await request(app).get('/ipfs/QmVickyWasHereDir/original.jpg')
+    await request(app).get('/content/QmVickyWasHere')
+    await request(app).get('/content/QmVickyWasHereDir/original.jpg')
+
+    const resp = await request(app)
+      .get('/prometheus_metrics_worker')
+      .expect(200)
+
+    assert.ok(
+      resp.text.includes(
+        'audius_cn_stream_content_duration_seconds_bucket{le="0.1",result="abort_bad_cid",mode="default"}'
+      )
+    )
+
+    assert.ok(
+      resp.text.includes(
+        'audius_cn_stream_content_dir_duration_seconds_bucket{le="0.1",result="abort_cid_not_found_in_db"}'
+      )
```

Comment:
curious why is the stream_content_duration record QmVickyWasHere as bad cid but dir metric does not?

Reply:
there is no cid check in the getdircid flow